www/js/app.js

```diff
@@ -145,6 +145,32 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         document.getElementById('searchArticles').click();
         return false;
     });
+     //Handle Home key press to focus #prefix
+     $(window,document).on('keydown', function(e) {
+         //if home key is pressed then
+         if(/^Home/.test(e.key)) {
+             //scroll to top of #search-article section since scroll bar is associated with it and not to window or document
+             $('#search-article').scrollTop(0);
+             $('#prefix').focus();
+         }
+     });
+     //Handle Home key event when focus is inside iframe #articleContent
+     var iframe= document.getElementById('articleContent');
+     //onload event is not fired in chrome for iframe if content is from local storage
+     //to compensate for onload event setTimeout can be used
+     setInterval(function (){
+         let iframewindow= iframe.contentWindow? iframe.contentWindow : iframe.contentDocument.defaultView;
```

Comment:
I suppose this is necessary to handle some old versions of IE?
We only support IE11 for kiwix-js, not earlier versions.
So it's probably enough to use directly `.contentWindow` (like it's done in `findDirEntryFromDirEntryIdAndLaunchArticleRead` function, for example)

Reply:
Thanks for clarifying. New commit uses `.contentWindow` .